app/models/user.rb

```diff
@@ -24,7 +24,8 @@ class User < ApplicationRecord
             presence: true,
             length: { within: 6..80 },
             on: :update,
-            if: :password_present?
+            if: :password_present?,
+            confirmation: true
```

Comment:
this could probably do with a model test:

```patch
From 50634d918f0715a5357eefce00800170c112eca0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?St=C3=A9phane=20Maniaci?= <stephane.maniaci@gmail.com>
Date: Wed, 17 Jun 2020 16:28:39 +0100
Subject: [PATCH] fix: enforce password confirmation at the user model level

---
 spec/models/user_spec.rb | 14 +++++++++++++-
 1 file changed, 13 insertions(+), 1 deletion(-)

diff --git a/spec/models/user_spec.rb b/spec/models/user_spec.rb
index 2ba1aebe..2332f6fa 100644
--- a/spec/models/user_spec.rb
+++ b/spec/models/user_spec.rb
@@ -3,7 +3,7 @@ describe User do
 
   describe "validation" do
     let(:user) { create(:user) }
-
+    let(:pw) { "this is 333 a strong !!! password" }
     it "has a valid factory user" do
       expect(user).to be_valid
     end
@@ -15,6 +15,18 @@ describe User do
         expect(user).not_to be_valid
       end
     end
+
+    it 'accepts a strong password' do
+      user.update(password: pw)
+
+      expect(user).to be_valid
+    end
+
+    it 'checks for a password confirmation field' do
+      user.update(password: pw, password_confirmation: pw.reverse)
+
+      expect(user).not_to be_valid
+    end
   end
 
   it { is_expected.to have_many(:organisations).through(:memberships) }
-- 
2.26.2
```